controller.go

```diff
@@ -778,6 +778,8 @@ func (c *controller) addNetwork(n *network) error {
 		return err
 	}
 
+	n.startResolver()
+
```

Comment:
Yes. Windows DNS resolver inside the container would send out a new query for every endpoint the container is connected to so this should not be an issue. Every network connected to the container will get a query and will be able to resolve the name.